### PR TITLE
Migrate shortlinks from Google Spreadsheet to GitHub CSV

### PR DESCRIPTION
**Pull request by patcon**
Ready for review.

I've opted to keep instructions for "managing shortlinks" in with the spreadsheet, and the little helper gif in the manual: https://github.com/hyphacoop/handbook/pull/11

Open to alternatives, just have feels on instructions living too far from the code on which those instructions depend? I dunno

### CODE REVIEW

File: shortlinks.csv [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+leads,https://docs.google.com/spreadsheets/d/1LyElXvLj7ASclSPQqcgOnuEWqBWBStilEfOhIosZzhA/edit#gid=0
+opportunities,https://link.hypha.coop/leads
+conversions,https://docs.google.com/spreadsheets/d/1LyElXvLj7ASclSPQqcgOnuEWqBWBStilEfOhIosZzhA/edit#gid=30829554
+slogans,https://hackmd.io/s/HkRaA_woE
+chants,https://hackmd.io/s/Bk-BO6viN
+twitter,https://twitter.com/hyphacoop
+logs,https://circleci.com/gh/hyphacoop/worker-coop-scripts/tree/master
+interviews,https://docs.google.com/spreadsheets/d/1vjnT0afFh-Ss1BCYZg7nuLyjwZqzRLsSzkhej_OdoFo/edit#gid=0
+schedules,https://www.when2meet.com/?7708144-gVsJ3
+donations,https://docs.google.com/spreadsheets/d/1DpnzCpXWgQr66nFJFIlpa5XSBrt8IfZaLVZH0_q6kcA/edit#gid=2079904514
```
Comment (dcwalk) [inline shortlinks.csv:31]: How will this work once we get rid of the shortlink google sheet?

**Review by dcwalk — APPROVED** [review]
Think this seems good, just curious about one link but shouldn't be a blocker :dancer:

File: shortlinks.csv (hunk 1 above)
Comment (patcon) [reply shortlinks.csv:31]: how do you mean? Won't it still simply point to another sheet? Our CRM and other scratch space stuff (like donations) is still tentatively in GDocs, right?

Comment (patcon) [reply shortlinks.csv:31]: Oh, I understand. It's just another worksheet in that same google spreadsheet. So we can remove the shortlinks sheet, and rename the overall gsheet, and the link is unaffected.

Comment (dcwalk) [reply shortlinks.csv:31]: sounds good!

**Comment by dcwalk** [comment]
Let's not stay blocked, I'm going to merge this! Thanks @patcon

**Pull request merged**